### DISCUSSION TITLE
GPU Recommendation (linux)

### DISCUSSION THREAD

**Discussion by BillyCroan** [openai/whisper#1527] [General]
I need to run Whisper in Linux, on my own GPU. I’ve found it quite useful on my Debian laptop, but without an FPGA it just kills the battery. I have a 4U server in colocation with some empty pcie slots running CentOS 8 (yuck). I’d be happy to reload that with Debian if necessary but I need to keep it GNU+Linux.

What is a good GPU recommendation I should get? I don’t think I want to get the latest and greatest gpu. I’d like to get something a generation or two old, used, to save money and reduce the e-waste in this world. My CPU is a Nehalem as well, if that is important. (I saw a warning about rocm not supporting cpus prior to Haswell.)

So. What do you recommend?

**Comment by phineas-pta** [comment]
tesla k80, super cheap for 24gb vram

**Reply by dgoryeo** [reply]
@phineas-pta what do you think about tesla P40? The specs look quite good but I keep reading that it underperforms for fp16.
